presto-orc/src/main/java/com/facebook/presto/orc/writer/SegmentedSliceBlockBuilder.java

```diff
@@ -71,8 +71,9 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(SegmentedSliceBlockBuilder.class).instanceSize();
 
-    private final DynamicSliceOutput openSliceOutput;
+    private final int expectedBytes;
 
+    private DynamicSliceOutput openSliceOutput;
```

Comment:
Can we line up the stack variable declaration with their assignments in the constructor? For example, if `expectedBytes` is declared before `offsets`, then it should be assigned before `offsets` as well. Same for all other variables.